### PR TITLE
Mobile debugging

### PR DESCRIPTION
**Pull request by canvaspixels**
This PR adds a bit of extra debugging when locating elements. Add the debug env var before your run like this: `DEBUG=1 npx ct` to see the logs. This PR also removes the auto run of the setup scripts. They now need to be run manually rather than on postinstall. See https://courgette-testing.com/getting-started

### CODE REVIEW

File: scripts/installFFDriver.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const { runScript } = require('./util');
+
+const setupDrivers = async function () {
+  try {
+    await runScript('./node_modules/protractor/bin/webdriver-manager', 'update --chrome=false'.split(' '));
+    console.log('FirefoxDriver Installed');
+  } catch (err) {
+    console.log(' ');
+    console.log('!!!!!!!!!!!-----------IMPORTANT----------!!!!!!!!!!!!!!!');
+    console.log('It looks like it hasn’t install properly, you may be behind a corporate proxy. You may have to add the --proxy flag to webdriver-manager in your package json.');
```
Comment (gminova) [inline scripts/installFFDriver.js]: hasn't installed*

Comment (canvaspixels) [reply scripts/installFFDriver.js]: ?

Comment (gminova) [reply scripts/installFFDriver.js]: It's a grammar error.
```
It looks like it hasn’t install properly
```
should be
```
It looks like it hasn’t installed properly
```

Comment (canvaspixels) [reply scripts/installFFDriver.js]: Oh wow, I looked at that like 3 times (under the weather)

**Pull request merged**